File: app/src/main/java/com/example/brochures/network/parsingobjects/Item.java
package com.example.brochures.network.parsingobjects;

public class Item {

    @com.squareup.moshi.Json(name = "content")
    private Content__2 content;
    @com.squareup.moshi.Json(name = "externalTracking")
    private ExternalTracking externalTracking;

    public Content__2 getContent() {
        return content;
    }

    public void setContent(Content__2 content) {
        this.content = content;
    }

    public ExternalTracking getExternalTracking() {
        return externalTracking;
    }

    public void setExternalTracking(ExternalTracking externalTracking) {
        this.externalTracking = externalTracking;
    }

}

File: app/src/main/java/com/example/brochures/network/parsingobjects/Item.kt
package com.example.brochures.network.parsingobjects

import com.squareup.moshi.Json

class Item {
    @Json(name = "content")
    var content: Content__2? = null

    @Json(name = "externalTracking")
    var externalTracking: ExternalTracking? = null
}